Documentation/subcommands/run.md

```diff
@@ -44,6 +44,14 @@ In the following example, the CPU isolator is defined to 750 milli-cores and the
 # rkt run coreos.com/etcd:v2.0.0 --cpu=750m --memory=128M
 ```
 
+## Overriding User/Group
+
+Application images must specify the username/group or the UID/GID the app is to be run as as specified in the [Image Manifest Schema](https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema). The user/group can be overridden by rkt using the `--user` and `--group` flags:
```

Comment:
We seem to have dropped the "one sentence, one line" policy in favor of "whole paragraph, one line".